en/components/bullet-graph.md

```diff
@@ -12,100 +12,63 @@ For scenarios where no target value exists or is irrelevant you should use the [
+## Legend
 
-Binding:
+The Bullet Graph comes with a Legend that can be adjusted to match the selected Series Amount. Legend items can be removed by setting the corresponding Series to No symbol or added by finding it in the Overrides/Charts/Legend/Serie. Also there you can change the icon type between Square, Line or Circle.
 
-<img class="responsive-img" src="../images/linear_gauge_bind_min.png"/>
+<img class="responsive-img" src="../images/bullet_graph_legend.png" srcset="../images/bullet_graph_legend@2x.png 2x" />
 
-No binding:
+## Tooltip
 
-<img class="responsive-img" src="../images/linear_gauge_nobind_min.png"/>
+The Bullet Graph comes with an override for the tooltip visibility. Setting in to No Symbol hides it.
 
-### Maximum Value
+<img class="responsive-img" src="../images/bullet_graph_tooltip-off.png" srcset="../images/bullet_graph_tooltip-off@2x.png 2x" />
+<img class="responsive-img" src="../images/bullet_graph_tooltip-on.png" srcset="../images/bullet_graph_tooltip-on@2x.png" 2x/>
 
-This override controls the maximum value displayed by the Bullet Graph. It supports [data binding](../codegen/data-binding.md) as well as direct setting of a number. If a binding is used the bound property must result in a number at runtime. If no binding is used the value must be a number. If the value provided is not in the correct format the Bullet Graph will be generated without a maximum value set and the control default will be used.
-
-Binding:
-
-<img class="responsive-img" src="../images/linear_gauge_bind_max.png"/>
-
-No binding:
-
-<img class="responsive-img" src="../images/linear_gauge_nobind_max.png"/>
-
-### Ranges Amount
-
-This override controls how many ranges are displayed within the Bullet Graph. During code generation it will create `<igx-linear-graph-range>` elements which are what represents a single range. This range element has `startValue`, `endValue` and `brush` properties which are used to define where the range starts and ends and what color it is. These are all automatically generated for you such that regardless of what the Minimum Value and Maximum Value overrides are set to you will always get the same appearance as what is displayed in the Sketch.
-
-You may not want the ranges to use the auto-generated start and end values, however, in that case you will need to manually update the ranges in the generated HTML.
-
-### ValueDataProperty
+## Ranges Amount
 
-This override controls the current value of the Bullet Graph.
+The Bullet Graph comes with three preset range configurations: **two**, three, and four ranges.
 
-<img class="responsive-img" src="../images/bullet_graph_value.png"/>
+<img class="responsive-img" src="../images/bullet_graph_two_ranges.png" srcset="../images/bullet_graph_two_ranges@2x.png 2x" />
```

Comment:
Here in the images show only the range portions

Reply:
I am not sure what do you mean about this change. Can you please elaborate?

Reply:
Three images that would look like this in the final document
<img width="550" alt="Screenshot 2021-01-15 at 11 00 18" src="https://user-images.githubusercontent.com/1702064/104704140-f1aaba80-5720-11eb-86d0-d152fadfc73c.png">

Reply:
Ignore the dark background, I made a screenshot but I think you will get my point